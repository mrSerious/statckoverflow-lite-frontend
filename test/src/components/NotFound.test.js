import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../../../src/components/404/NotFound';

test('NotFound snapshot test', () => {
  const component = shallow(<NotFound />);
  expect(component).toMatchSnapshot();
});
