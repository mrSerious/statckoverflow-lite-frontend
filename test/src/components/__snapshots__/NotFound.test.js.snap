// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound snapshot test 1`] = `
<div>
  <h1>
    404 Page Not Found!
  </h1>
</div>
`;
